app/helpers/application_helper/button/rbac_user_copy.rb

```diff
@@ -0,0 +1,8 @@
+class ApplicationHelper::Button::RbacUserCopy < ApplicationHelper::Button::Basic
+  needs :@record
+
+  def disabled?
+    @error_message = _('User [Administrator] can not be copied') if @record.super_admin_user?
```

Comment:
Maybe should use different error message. Maybe something like: 'User with [Administrator] role can not be copied' It's little bit confusing when clicking in UI. But otherwise code looks good, and in UI It works same as before.